go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/diktyo-io/appgroup-api v0.0.9-alpha
 	github.com/diktyo-io/networktopology-api v0.0.8-alpha
 	github.com/dustin/go-humanize v1.0.0
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.7
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.12
 	github.com/k8stopologyawareschedwg/podfingerprint v0.1.1
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/paypal/load-watcher v0.2.2
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
-	gonum.org/v1/gonum v0.6.2
+	gonum.org/v1/gonum v0.12.0
```

Comment:
could you please add a couple line in the commit message to explain why we need the dep bump? (not obvious from the next commits)

Reply:
Commit reworded